introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi I am V. and pretty friggin awesome and so are you.  My favorite Code School path has been JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?